### PR TITLE
OSDOCS-8457:adds 4.12.42 to RNs

### PR DESCRIPTION
**Pull request by JoeAldinger**
Version(s):
4.12

Issue:
https://issues.redhat.com/browse/OSDOCS-8457

Link to docs preview:
https://67570--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes#ocp-4-12-42

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -4254,8 +4254,42 @@ You can view the container images in this release by running the following comma
...
+
+===== APIServer.config.openshift.io is now tracked by Insights Operator
+
+After running the Insights Operator, a new file is now available in the archive in the path `config/apiserver.json` with the information about the audit profile for `APIServer.config.openshift.io`.
+
+Access to audit profiles help you to understand what audit policy is common practice, what profiles are most commonly used, what differences there are between industries, and what kind of customization is applied.
+
+[id="ocp-4-12-42-bug-fixes"]
+==== Bug fixes
+* Previously, the Cluster Version Operator (CVO) did not reconcile `SecurityContextConstraints` (SCC) resources as expected. The CVO now properly reconciles the `volumes` field in the `SecurityContextConstraints` resources towards the state defined in the release image. User modifications to system SCC resources are tolerated. For more information on how SCC resources can impact updating, see link:https://access.redhat.com/solutions/7033949[Resolving Detected modified SecurityContextConstraints update gate before upgrading to 4.14
```
Comment (tmalove) [inline release_notes/ocp-4-12-release-notes.adoc:4287]:
```suggestion
* Previously, the Cluster Version Operator (CVO) did not reconcile `SecurityContextConstraints` (SCC) resources as expected. The CVO now properly reconciles the `volumes` field in the `SecurityContextConstraints` resources towards the state defined in the release image. User modifications to system SCC resources are tolerated. For more information about how SCC resources can impact updating, see link:https://access.redhat.com/solutions/7033949[Resolving Detected modified SecurityContextConstraints update gate before upgrading to 4.14
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#i

File: release_notes/ocp-4-12-release-notes.adoc [hunk 2]
```diff
@@ -4254,8 +4254,42 @@ You can view the container images in this release by running the following comma
...
 ==== Updating
 
 To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+[id="ocp-4-12-42"]
+=== RHSA-2023:6276 - {product-title} 4.12.42 bug fix and security update
+
+Issued: 2023-11-08
+
+{product-title} release 4.12.42, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:6276[RHSA-2023:6276] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:6278[RHSA-2023:6278] advisory.
```
Comment (tmalove) [inline release_notes/ocp-4-12-release-notes.adoc:4267]: Checking that this is expected behavior - the `6276` errata resolves, but the `6278` does not. Should they both be the same?

**Review by tmalove — APPROVED** [review]

File: release_notes/ocp-4-12-release-notes.adoc (hunk 2 above)
Comment (JoeAldinger) [reply release_notes/ocp-4-12-release-notes.adoc:4267]: Typo on my part. RHBA not RHSA

**Pull request merged**